### PR TITLE
SDKS-7151: Instant feature flags update integration tests

### PR DESCRIPTION
**Pull request by javrudsky**
# iOS SDK

## What did you accomplish?
- Triggering  sdk_udpated event when IFFU
- Added integration tests

### CODE REVIEW

File: Split/Network/Streaming/DefaultSseNotificationParser.swift [hunk 1]
```diff
@@ -50,6 +50,7 @@ class DefaultSseNotificationParser: SseNotificationParser {
             return IncomingNotification(type: type, channel: rawNotification.channel,
                                         jsonData: rawNotification.data, timestamp: rawNotification.timestamp ?? 0)
         } catch {
+            print("ERROR: \(error.localizedDescription)")
```
Comment (EmilianoSanchez) [inline Split/Network/Streaming/DefaultSseNotificationParser.swift:53]: `print` or `Logger.e` ?

File: SplitTests/Fake/Streaming/SynchronizerStub.swift [hunk 2]
```diff
@@ -184,6 +184,11 @@ class SynchronizerStub: Synchronizer {
         notifyMySegmentsUpdatedCalled = true
     }
 
+    var notifyFeatureFlagsUpdatedCalled = true
```
Comment (EmilianoSanchez) [inline SplitTests/Fake/Streaming/SynchronizerStub.swift:187]: `var notifyFeatureFlagsUpdatedCalled = false` ?

**Review by EmilianoSanchez — APPROVED** [review]

File: Split/Network/Streaming/DefaultSseNotificationParser.swift (hunk 1 above)
Comment (javrudsky) [reply Split/Network/Streaming/DefaultSseNotificationParser.swift:53]: Thanks doc! 
Fixed 👍

File: SplitTests/Fake/Streaming/SynchronizerStub.swift (hunk 2 above)
Comment (javrudsky) [reply SplitTests/Fake/Streaming/SynchronizerStub.swift:187]: Same here! Fixed!

**Pull request merged**